src/app/tests/suites/certification/Test_TC_SC_5_2.yaml

```diff
@@ -17,46 +17,141 @@ name: 19.1.2. [TC-SC-5.2] Receiving a group message - TH to DUT
+                      Subjects: [commissionerNodeId],
+                      Targets: null,
+                  },
+                  {
+                      FabricIndex: 1,
+                      Privilege: 3,
+                      AuthMode: 3,
+                      Subjects: [0x0103],
+                      Targets: null,
+                  },
+              ]
 
-          DUT supports the Identify cluster
-      disabled: true
+    - label: "TH sends KeySetWrite command with TH key"
+      cluster: "Group Key Management"
+      endpoint: 0
+      command: "KeySetWrite"
+      arguments:
+          values:
+              - name: GroupKeySet
+                value:
+                    {
+                        GroupKeySetID: 0x01a3,
+                        GroupKeySecurityPolicy: 0,
+                        EpochKey0: "0xd0d1d2d3d4d5d6d7d8d9dadbdcdddedf",
+                        EpochStartTime0: 2220000,
+                        EpochKey1: "0xd1d1d2d3d4d5d6d7d8d9dadbdcdddedf",
+                        EpochStartTime1: 2220001,
+                        EpochKey2: "0xd2d1d2d3d4d5d6d7d8d9dadbdcdddedf",
+                        EpochStartTime2: 2220002,
+                    }
 
-    - label: "TH writes ACL entry by setting AuthMode as Group to DUT"
-      verification: |
-          As this step is executed in the TC-SC-5.1, we can skip this step during execution.
-      disabled: true
+    - label: "TH binds GroupId to GroupKeySet"
+      cluster: "Group Key Management"
+      endpoint: 0
+      command: "writeAttribute"
+      attribute: "GroupKeyMap"
+      arguments:
+          value: [{ FabricIndex: 1, GroupId: 0x0103, GroupKeySetID: 0x01a3 }]
 
-    - label:
-          "TH sends a multicast Identify command with the IdentifyTime set to
-          0x0078 (120s) to DUT with GroupID and PIXIT.G.ENDPOINT"
-      PICS: I.S.C00.Rsp
-      verification: |
-          ./chip-tool identify identify 0x0078 0xffffffffffff0001 1
+    - label: "TH sends RemoveAllGroups command"
+      command: "RemoveAllGroups"
 
-          Verify command send  to group on the TH Log:
+    - label: "TH sends AddGroup command"
+      command: "AddGroup"
+      arguments:
+          values:
+              - name: GroupID
+                value: 0x0103
+              - name: GroupName
+                value: "Test Group"
 
-          [1657785273.973231][1618:1623] CHIP:DL: HandlePlatformSpecificBLEEvent 32784[1657785273.973291][1618:1623] CHIP:TOO: Sending command to group 0x1
-          [1657785273.973316][1618:1623] CHIP:TOO: Sending cluster (0x00000003) command (0x00000000) on Group 1
-      disabled: true
+    - label: "TH sends ViewGroup command using group messaging"
+      PICS: G.S.F00
+      command: "ViewGroup"
+      groupId: 0x0103
+      arguments:
+          values:
+              - name: GroupID
+                value: 0x0103
+      response:
+          values:
+              - name: Status
+                value: 0
+              - name: GroupId
```

Comment:
I would expect the name here (and similar places) to be "GroupID"...

Reply:
changed in next. Doesn't affect the generated files since zap does a case change.